src/logics/LALogic.h

```diff
@@ -49,8 +49,8 @@ class LALogic: public Logic
 public:
     LALogic();
     ~LALogic() { for(int i = 0; i < numbers.size(); ++i) {delete numbers[i];}}
-    virtual bool     isBuiltinFunction(SymRef sr) const override;
-    virtual PTRef    insertTerm       (SymRef sym, vec<PTRef>& terms, char** msg) override;
+    bool             isBuiltinFunction(SymRef sr) const override;
+    PTRef            insertTerm       (SymRef sym, vec<PTRef>& terms) override;
```

Comment:
I would keep the virtual keyword here, since it is present also for other overriden methods (below).

Reply:
Or we could remove it from below as well, when it's redundant?

Reply:
Or that.
But this is one of the places where I consider something redundant to be useful. But maybe I am just too used to virtual functions being marked as `virtual` :)